server/game/cards/08-MotC/KitsukiJusai.js

```diff
@@ -0,0 +1,29 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class KitsukiJusai extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            title: 'Put a fate from your opponent pool on an unclaimed ring',
+            when: {
+                onHonorDialsRevealed: (event, context) =>
+                    context.player.opponent &&
+                    context.player.honorBid === context.player.opponent.honorBid &&
+                    context.player.opponent.fate > 0
+            },
+            gameAction: AbilityDsl.actions.selectRing(context => ({
+                activePromptTitle: 'Choose an unclaimed ring to move fate to',
+                ringCondition: ring => ring.isUnclaimed() && ring !== context.ring,
```

Comment:
what is the `context.ring` check for?  This ability doesn't target a ring, so `context.ring` should always be `undefined`...

Reply:
Ah you're right, leftover from a copy :D